migrations/20230213080003-fast-balance-update.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface) {
+    await queryInterface.sequelize.query(`DROP VIEW IF EXISTS "CurrentCollectiveBalance"`);
+
+    await queryInterface.sequelize.query(`DROP MATERIALIZED VIEW IF EXISTS "CollectiveBalanceCheckpoint"`);
+
+    await queryInterface.sequelize.query(`
+      CREATE MATERIALIZED VIEW IF NOT EXISTS "CollectiveBalanceCheckpoint" AS (
+        WITH "LatestTransactions" AS (
+          SELECT "CollectiveId", MAX("createdAt") AS "createdAt" FROM "TransactionBalances" GROUP BY "CollectiveId"
+        )
+        SELECT *
+        FROM "TransactionBalances"
+        WHERE id IN (
+          SELECT MAX(tb."id") FROM "TransactionBalances" tb
+          INNER JOIN "LatestTransactions" lt ON lt."CollectiveId" = tb."CollectiveId" AND lt."createdAt" = tb."createdAt"
+          GROUP BY tb."CollectiveId"
+        )
+       )
+    `);
+
+    await queryInterface.sequelize.query(`
+      CREATE UNIQUE INDEX CONCURRENTLY IF NOT EXISTS "collective_balance_checkpoint__collective_id"
+      ON "CollectiveBalanceCheckpoint"("CollectiveId")
+    `);
```

Comment:
This index could also be added to `CollectiveTransactionStats(id)` on the other migration.

Reply:
Good catch, we used to have it, then lost it in a former migration, had to add it again.

We were about to loose it again.